covid_moonshot/analysis/__init__.py

```diff
@@ -18,8 +18,9 @@ def get_run_analysis(
         raise ValueError(f"Failed to analyze solvent: {e}")
 
     binding = Binding(
-        delta_f=solvent_phase.delta_f - complex_phase.delta_f,
-        ddelta_f=(complex_phase.ddelta_f ** 2 + solvent_phase.ddelta_f ** 2) ** 0.5,
+        delta_f=solvent_phase.free_energy.delta_f - complex_phase.free_energy.delta_f,
+        ddelta_f=(complex_phase.free_energy.ddelta_f ** 2 + solvent_phase.free_energy.ddelta_f ** 2)
```

Comment:
The `dX` quantities should represent the standard error, so you need to take the sqrt after adding in quadrature.
This should be
```python
        ddelta_f=np.sqrt(complex_phase.free_energy.ddelta_f ** 2 + solvent_phase.free_energy.ddelta_f ** 2)
```

Reply:
Oh, I see you have `** 0.5` on the next line. Let's use `np.sqrt` instead so it's obvious that this isn't especially hard to read.

Reply:
Agreed on `np.sqrt` for readability! I hadn't noticed that the formatter moved `** 0.5` down a line.

Reply:
Changed in 4c57430